src/components/usersComponent.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+
+import Userdetail from "./userdetailComponent";
+
+class Users extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Maybe we can do this outside of the constructor?
We should decide whether we want to do this everywhere in the constructor or outside the constructor for consistency.

Reply:
Inside the constructor is most commonly used, I suggest we do the same